static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v3.js

```diff
@@ -49,80 +51,79 @@ define([
     ExpandableV3.hasScrollEnabled = !detect.isIOS() && !detect.isAndroid();
 
     ExpandableV3.prototype.updateBgPosition = function () {
-        var adHeight, inViewB, inViewT, topCusp, bottomCusp, bottomScroll, topScroll;
-        fastdom.read(function () {
-            adHeight = (this.isClosed) ?
+        var scrollY = window.pageYOffset,
+            viewportHeight = bonzo.viewport().height,
+            adSlotTop = this.$adSlot.offset().top,
+            that = this;
+
+        var adHeight = (this.isClosed) ?
                 this.closedHeight : this.openedHeight;
-            inViewB = ((window.pageYOffset + bonzo.viewport().height) > this.$adSlot.offset().top);
-            inViewT = ((window.pageYOffset - (adHeight * 2)) < this.$adSlot.offset().top + 20);
-            topCusp = (inViewT &&
-                ((window.pageYOffset + (bonzo.viewport().height * 0.4) - adHeight) > this.$adSlot.offset().top)) ?
+        var inViewB = ((scrollY + viewportHeight) > adSlotTop);
+        var inViewT = ((scrollY - (adHeight * 2)) < adSlotTop + 20);
+        var topCusp = (inViewT &&
+                ((scrollY + (viewportHeight * 0.4) - adHeight) > adSlotTop)) ?
                 'true' : 'false';
-            bottomCusp = (inViewB &&
-                (window.pageYOffset + (bonzo.viewport().height * 0.5)) < this.$adSlot.offset().top) ?
+        var bottomCusp = (inViewB &&
```

Comment:
Another question for whomever wrote this code in the first place

```
var bottomCusp = (..something..) ? 'true' : 'false';
if (bottomCusp === 'true') {}
```

It's a bit convoluted.

This one looks better
```
var bottomCusp = !!(..something..);  // Bolean
if (bottomCusp) {}
```